ansible/roles/pmix/defaults/main.yaml

```diff
@@ -0,0 +1,3 @@
+---
+
+pmix_version: 4.2.9
```

Comment:
I believe that this be overridden by the user in a solution like:

https://github.com/GoogleCloudPlatform/hpc-toolkit/blob/08ae77e52e4cab73a1b5ebb2144751c7dd752225/community/examples/hpc-build-slurm-image.yaml

So long as the version exists, correct?

Reply:
You mean if they put it in that variable file or at the cli command?  Yes, I believe that will override it. 

Given that, we could add in a check for pmix > 2 because that's the minimum for modern slurm.

Reply:
I do not think that is strictly necessary given PMIx v2's age.